Cmdline/app.config

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm not familiar with `app.config` files. How do they get used? I'm mostly concerned about having files that our build system may ignore.